gh-pages/baseline-in-the-wild.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: baseline-in-the-wild
+---
+
+# Baseline in the wild
+
+You can find Baseline in the following places. If you spot somewhere else using Baseline, or are using Baseline on your own site, or to convey compat information in your library, create a PR to add it to the list.
+
+## From browser vendors
+
+- [Baseline on MDN (Firefox)](https://developer.mozilla.org/en-US/blog/baseline-unified-view-stable-web-features/)
+- [Baseline on web.dev (Google)](https://web.dev/baseline/)
+<!-- - Baseline on MSDN (Microsoft) -- Couldn't find appropriate link -->
```

Comment:
@captainbrosset - is there a good site/page I should like to for Microsoft/Edge?

Reply:
No there isn't a Microsoft Edge page about Baseline for now.